src/assets/i18n/nl.json

```diff
@@ -64,6 +64,6 @@
   "CODE": "Code",
   "BLOCKS": "Blokken",
   "DIRECT_TO_CODE_EDITOR": "Direct naar Code-scherm",
-  "CONFIRMEDITORCHANGE_TITLE": "Onopgeslagen wijzigingen",
-  "EDITORCHANGEINSTRUCTIONS": "Je hebt onopgeslagen wijzigingen. Weet je zeker dat je terug wilt naar de blokken editor? Je wijzigingen zullen verloren raken!"
+  "CONFIRMEDITORCHANGE_TITLE": "Wijzigingen opslaan?",
+  "EDITORCHANGEINSTRUCTIONS": "Je hebt onopgeslagen wijzigingen. \rAls je nu naar het Blokken-scherm gaat, ben je die kwijt."
```

Comment:
There is a carriage return character (\r) in here that is not in the en.json. Is that intentional?

Reply:
No! I was experementing with a way to start on a new line...but it didn't work out.
And apparently i didn't remove one of the tries...